modules/learn/pages/clusters-and-availability/hard-failover.adoc

```diff
@@ -21,6 +21,29 @@ Hard failover can be initiated by means of:
 
 For further information on initiating hard failover, see xref:manage:manage-nodes/failover-hard.adoc[Perform Hard Failover].
 
+[#safe-and-unsafe]
+== Hard-Failover Modes: Safe and Unsafe
+
+Either of the following two _modes_ can be specified for hard failover:
+
+* _Safe_ (the default mode).
+When hard failover has been initiated, it is continued by Couchbase Server only in the following circumstances:
+
+** Data-loss cannot result.
```

Comment:
Actually not true. This needs to be removed. Graceful failover should not result in data loss. Regular hard failover will likely result in data loss. Unsafe hard failover is hard failover where the cluster manager can contact only a minority of nodes. 

The problem with this is that simultaneously another admin could be failing over the minority of nodes from the other side of the network partition. We have no way of stopping this and if these two hard failovers were to happen simultaneously the cluster configuration would be entirely corrupt when the network partition heals.

So in unsafe failover we require the administrator to tell us that no such failover is being performed from the other side of the partition and that the nodes being failed over are never going to "come back"